### PR TITLE
server, schedule: limiter record operator count group by store

### PR DESCRIPTION
**Pull request by Connor1996**
limiter records operator count grouping by store

### CODE REVIEW

**Comment by siddontang** [comment]
please add a description of the PR.

File: server/coordinator.go [hunk 1]
```diff
@@ -520,7 +518,7 @@ func (c *coordinator) removeOperator(op *schedule.Operator) {
 func (c *coordinator) removeOperatorLocked(op *schedule.Operator) {
 	regionID := op.RegionID()
 	delete(c.operators, regionID)
-	c.limiter.UpdateCounts(c.operators)
+	c.limiter.RemoveOperator(op)
```
Comment (nolouch) [inline server/coordinator.go]: refer to https://github.com/pingcap/pd/pull/874, we meet some not synchronized case so change to` updatecounts`.

Comment (Connor1996) [reply server/coordinator.go]: I have noticed that. But now we have a background goroutine to check all operators, all path will reach RemoveOperator() or AddOperator(). it's unlikely not synchronized.

File: server/schedule/operator.go [hunk 2]
```diff
@@ -36,6 +36,7 @@ type OperatorStep interface {
 	fmt.Stringer
 	IsFinish(region *core.RegionInfo) bool
 	Influence(opInfluence OpInfluence, region *core.RegionInfo)
+	InvolvedStores(region *core.RegionInfo) []uint64
```
Comment (disksing) [inline server/schedule/operator.go]: How about `AffectStores`?

File: server/coordinator.go (hunk 1 above)
Comment (disksing) [reply server/coordinator.go]: I vote for update limiter using all operators too.

File: server/schedule/operator.go (hunk 2 above)
Comment (disksing) [reply server/schedule/operator.go]: It's kind weird to have a `region` parameter here. I think you can add a `OriginLeader` field to operators that need original leader id to return affect stores.

**Comment by Connor1996** [comment]
@disksing @nolouch PTAL

**Pull request closed without merging**